main.go

```diff
@@ -137,3 +160,23 @@ func checkOSSIndex(purls []string, packageCount int) {
 		os.Exit(count)
 	}
 }
+
+func auditWithIQServer(purls []string, applicationID string, config configuration.Configuration) {
+	res := iq.AuditPackages(purls, applicationID, config)
+
+	fmt.Println()
+	if res.IsError {
+		fmt.Println("There was an error with your request to Nexus IQ Server", res.ErrorMessage)
+		os.Exit(1)
+	}
+
+	if res.PolicyAction != "Failure" {
+		fmt.Println("Wonderbar! No policy violations reported for this audit!")
+		fmt.Println("Report URL: ", res.ReportHTMLURL)
+		os.Exit(0)
+	} else {
+		fmt.Println("Sonabot here, you have some policy violations to clean up!")
```

Comment:
Why the reference to Sonabot?? nancy is not Sonabot so is that an IQ server reference??

Reply:
We had an account back in the day called "Sonabot", I used that phrase in the work we did on `jake` and `auditjs` mostly to be consistent but let me switch it to `nancy` :)

Reply:
In: 772d557